### PR TITLE
Final edits

### PR DESCRIPTION
**Pull request by rogrange**
Final edits from Phil for the containerized installation guide.

### CODE REVIEW

File: downstream/modules/platform/proc-preparing-the-rhel-host-for-containerized-installation.adoc [hunk 1]
```diff
@@ -23,6 +23,18 @@ rhel-9-for-x86_64-baseos-rpms                              Red Hat Enterprise Li
...
+
+.Using BIND DNS
+
+To configure DNS using BIND refer to link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_networking_infrastructure_services/assembly_setting-up-and-configuring-a-bind-dns-server_networking-infrastructure-services[Chapter 1. Setting up and configuring a BIND DNS server Red Hat Enterprise Linux 9].
+
+.Optional
+
+To have the installer automatically pick up and apply your AAP subscription manifest license, then use this guide to generate a manifest file which can be downloaded for the installer: link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_operations_guide/assembly-aap-obtain-manifest-files[Chapter 2. Obtaining a manifest file Red Hat Ansible Automation Platform 2.].
```
Comment (hherbly) [inline downstream/modules/platform/proc-preparing-the-rhel-host-for-containerized-installation.adoc:40]: AAP should be {PlatformNameShort} here

Comment (rogrange) [reply downstream/modules/platform/proc-preparing-the-rhel-host-for-containerized-installation.adoc:40]: Thanks! I knew I would miss one or two.  :-)

Comment (hherbly) [inline downstream/modules/platform/proc-preparing-the-rhel-host-for-containerized-installation.adoc:40]: Should the "then" be there? There isn't a step before this described in this sentence. I would remove it to make the sentence make grammatical sense, unless I'm misunderstanding something here.

Comment (rogrange) [reply downstream/modules/platform/proc-preparing-the-rhel-host-for-containerized-installation.adoc]: Nope. :-)

Comment (rogrange) [reply downstream/modules/platform/proc-preparing-the-rhel-host-for-containerized-installation.adoc]: Done.

**Review by hherbly — APPROVED** [review]
Looks like you've already addressed my comments, so LGTM!

**Pull request merged**